src/data/products.js

```diff
@@ -1,30 +1,39 @@
 import hoodie from '../assets/images/products/hoodie.png';
 import headphone from '../assets/images/products/headphone.png';
 import longCoat from '../assets/images/products/long-coat.png';
+import getRandomArbitraryNumber from '../utility';
+import { SIZES } from '../utility/appConstants';
```

Comment:
This utility is redundant. Add the raw array to each product item. We want data to be consistent across page refresh; at least until there's any user interaction.

Reply:
> Reason for the suggestion being, to write less code by using what's already available.

Regarding this, the point is, writing less code by using already available items is a good thing indeed but the component CHIP is not meant for this purpose, hence I suggest to use buttons here.